### ISSUE TITLE
Mailinglist

### ISSUE THREAD

**Issue by sebix** [winpython/winpython.github.io#38]
It is nearly impossible to find out how to subscribe to our mailinglist at googlegroups for non-gmail users. Therefore I would prefer to move the list to another provider, e.g. librelist.com

**Comment by stonebig** [comment]
other mailing list that I know seems to be on googlegroups:
- pydata
- spyder
- anaconda

**Comment by stonebig** [comment]
a side question: I see about 15% of linux downloads on Germany. Is it real users ? does WinPython works on "Wine" ? Does some users need that ?

**Comment by sebix** [comment]
> Other mailing list that I know seems to be on googlegroups:

Sorry, but this is not an argument.
> The mailing list is not that pretty active, so we may be bike-shedding here.

I do receive support requests via mail. This is an indicator, that we are doing something wrong.

> a side question: I see about 15% of linux downloads on Germany. Is it real users ? does WinPython works on "Wine" ? Does some users need that ?

No idea. We could add a link to Anaconda for Linux users. Winpython does have more recent software than distributions, but compiling yourself is also much much easier (just with pip install).

**Comment by stonebig** [comment]
I don't think we do anything more wrong than others on mailing list, there is maybe just better alternatives than mailing list nowodays:
- for user "undefined problems": stackoverflow, 
- for "not- english speaking users": localised langage chat platforms,
- for "users with well-defined feature request or solutions": github.

**Comment by hiccup7** [comment]
stackoverflow is an excellent resource.  I suggest adding to a link to http://stackoverflow.com/questions/tagged/python on https://winpython.github.io/

I notice that stackoverflow has no tag for WinPython.  Is it possible to add this?  You could encourage users to use the WinPython tag for questions specific to WinPython.

Today, I rely on the googlegroups release announcement for the SHA1 hash of the release downloads.  Could release announcements be put somewhere else so that googlegroups can be retired for WinPython?

**Comment by stonebig** [comment]
you need 1000 points to be able to suggest a new "tag", I'm 531.

**Comment by sebix** [comment]
I already thought about adding a tag for winpython some time ago.

Currently there are 428 questions on stackoverflow containing 'winpython', definitely more than when I thought about it last time. So it may be reasonable.
> You need 1000 points to be able to suggest a new "tag", I'm 531.

It requires 1500 reputation points.

**Comment by hiccup7** [comment]
I looked again at https://winpython.github.io/ and I see that the MD5 and SHA1 hash of releases is there already.  Sorry for not paying attention.

The main reason for me to use the googlegroups now is for WinPython announcements.  I like the highlights and summary of these.  But wouldn't a WinPython blog be better for announcements?

For me to post to googlegroups, I need to allow Google cookies and scripts.  If I allow these for all my web browsing, which is the most convenient, then Google tracks my browsing habits and sells my personal info.  Lately, when I try to log into my Google account to post on the WinPython group, Google accuses me of being a robot and asks for my landline or cellphone number to prove I am not a robot.  If I comply, I have given Google yet more personal info, and I have no control over to whom they sell this info.  Instead, I choose not to post on the WinPython googlegroup any more.  This means that WinPython users who ask for help there are not getting any help from me.

I notice that questions on the WinPython googlegroup sometimes don't get any answer, or a suggestion takes many days to arrive.  If I wanted a quick answer, I would go to stackoverflow or github instead.  New WinPython users may not know the options and feel frustrated with no response or a slow response on the googlegroup.  Some users are aware of how unpopular the WinPython googlegroup is, so they ask on stackoverflow too.  If their question is answered on stackoverflow, then posting to the googlegroup is a waste of time.

**Comment by sebix** [comment]
Using googlegroups without a google account is actually possible (that's what I did), but the function is very well hidden. I agree with you @hiccup7
